packages/flame_test/lib/src/flame_test.dart

```diff
@@ -11,6 +12,22 @@ extension FlameFinds on CommonFinders {
   }
 }
 
+extension FlameGameExtension on FlameGame {
+  /// Makes sure that the [component] is added to the tree if you wait for the
+  /// returned future to resolve.
+  Future<void> addUpdate(Component component) async {
```

Comment:
What if we call it ensureComponentAddition, or something like that?

Reply:
It's so long though, could we come up with a shorter name since it is going to be used everywhere?

Reply:
`ensureAdd` or `fullAdd` maybe? Since it is doing `add` which is acting with the component I don't think the component part in the name is needed.